From: Balint Reczey <balint.reczey@canonical.com>
Date: Thu, 5 Mar 2020 16:14:33 +0100
Subject: Find test fixtures in source directory
Forwarded: not-needed

---
 xbmc/test/TestUtils.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/xbmc/test/TestUtils.h b/xbmc/test/TestUtils.h
index b0dbcf6..0e46579 100644
--- a/xbmc/test/TestUtils.h
+++ b/xbmc/test/TestUtils.h
@@ -94,7 +94,7 @@ private:
   double probability;
 };
 
-#define XBMC_REF_FILE_PATH(s) CXBMCTestUtils::Instance().ReferenceFilePath(s)
+#define XBMC_REF_FILE_PATH(s) std::string("../" s)
 #define XBMC_CREATETEMPFILE(a) CXBMCTestUtils::Instance().CreateTempFile(a)
 #define XBMC_DELETETEMPFILE(a) CXBMCTestUtils::Instance().DeleteTempFile(a)
 #define XBMC_TEMPFILEPATH(a) CXBMCTestUtils::Instance().TempFilePath(a)
